handbook/company/pricing-features-table.yml

```diff
@@ -1133,23 +1109,10 @@
   usualDepartment: IT
   productCategories: [Device management]
   pricingTableCategories: [Device management]
-- industryName: API first by design
-  tier: Free
-  jamfProHasFeature: no
-  jamfProtectHasFeature: no
-  usualDepartment: IT
-  productCategories: [Device management]
-  pricingTableCategories: [Device management]
-- industryName: Targeted device scoping
-  tier: Premium
-  jamfProHasFeature: yes
-  jamfProtectHasFeature: yes
-  usualDepartment: IT
-  productCategories: [Device management]
-  pricingTableCategories: [Device management]
 - industryName: Application deployment
+  description: Deploy applications and security agents on macOS, iOS/iPadOS, Linux, Windows, and Android (coming soon) devices. Additionally, install macOS and iOS/iPadOS apps from the App Store.
```

Comment:
Consolidated `Application deployment,` `Deploy security agents on macOS, Windows, and Linux computers` and `Install Apple App Store apps through Volume Purchasing Program (VPP)` and updated the description/tooltip.